### DISCUSSION TITLE
error

### DISCUSSION THREAD

**Discussion by Odeneus** [beeb/pancaketrade#83] [Q&A]
Hi i am experiencing a small issue. Ik keep receiving this message now and then
2022-01-16 10:12:35 WARNING |Execution of job "TokenWatcher.monitor_price (trigger: interval[0:00:05], next run at: 2022-01-16 10:12:35 -03)" skipped: maximum number of running instances reached (1)
Nor sure what this means and why it keeps showing up, I have also noticed the limit sell not triggering even though price went above it more than once. Not sure if there is a connection between the warning message and the issue

**Comment by beeb** [comment]
This is only a warning. Normally, the script monitors the price of each token every 5 seconds. To avoid checking the price again while there are still pending requests, the script instead skips this iteration because price checking is already happening. This is because the RPC took more than 5 seconds to respond to the script's requests. It could be due to a slow RPC server, high number of tokens, or too low `monitor_interval`.

**Comment by beeb** [comment]
Please also share the configuration of your limit sell order (output message of the telegram command `/order 123` where 123 is your order number)

**Reply by Odeneus** [reply]
Hi thanks for explaining that. So the message has nothing to do with the order not filling?
This is the configuration of my limit sell order
BSCM - (#2) limit sell 💰
Amount: 7,316.4 BSCM
Price: > 0.0000765 BNB
Trailing stop loss 3% callback
Slippage: 3.00%
Gas: network default
Created: 2022-01-15 21:01

**Reply by beeb** [reply]
That's correct, the warning should not affect the order triggering. Since you have 3% callback rate, the order will only execute after the target price was exceeded **and then** dropped by at least 3% compared to the maximum. If you restart the bot while the order is tracking, it might lose track of the previous maximum value because it's only kept in memory. This is why the script should ideally be running continuously.

**Reply by Odeneus** [reply]
Ok i bought at $0.0338 and the price went up to $0.04353. I am not sure if i restarted the bot during the tracking period, but i will keep this in mind. I also am in the process in setting up a private node. I think it will run better on this

**Reply by beeb** [reply]
Also keep in mind that the price is being tracked in BNB at the moment. If the value of BNB increases, it doesn't mean the value of the token (in BNB) increases but you would see a higher dollar value.

**Reply by Odeneus** [reply]
Oh i see... are you planning on implementing busd tracking in the bot anytime soon?

**Reply by beeb** [reply]
This has been added since v0.7
